packages/ketchup/src/f-components/f-cell/f-cell.tsx

```diff
@@ -870,6 +870,9 @@ function setEditableCell(
                                 ? column.icon
                                 : null
                         }
+                        decimals={props.column.decimals}
+                        integers={props.column.integers}
+                        size={+props.column.size}
```

Comment:
column.size can be a value in px.
Was this taken into account?

e.g.: 100 -> 100px - this would translate into a textfield with 100 characters